### DISCUSSION TITLE
Mobile access?

### DISCUSSION THREAD

**Discussion by Yetiman82** [anapnoe/stable-diffusion-webui-ux#155] [General]
Hi all. I see in the description for this fork that it is mobile friendly. I can't find any info on how this works. Does it run on mobile using some kind of server on a pc?

**Comment by DrJKL** [comment]
If you're running with `--share` or `--listen` and can open the client on a Mobile device, the UI is responsive, making it a more pleasant experience than a desktop optimized page.

**Reply by Yetiman82** [reply]
Perfect, thank you!
